src/com/pentalog/pentastagiu/homework2/registration/Message.java

```diff
@@ -0,0 +1,40 @@
+package com.pentalog.pentastagiu.homework2.registration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Message {
+    private String description;
+    private String author;
+
+    public static List<String> messageList = new ArrayList<>();
```

Comment:
nu inteleg de ce ai avea nevoie de pastrarea unei liste a mesajelor aici + este adaugata dar nu este folosita nicaieri in afara de getter/setter